### PR TITLE
Improved Keras CIFAR10 example

### PR DESCRIPTION
**Pull request by stefannica**
The previous ML model was heavily overfitting during training.
Some improvements are made to the mode architecture to produce
better results:

* more convolution layers
* dropout layers

The improvements are based on this Machine Learning Mastery article:

https://machinelearningmastery.com/how-to-develop-a-cnn-from-scratch-for-cifar-10-photo-classification/

Other modifications:

* use the conda tensorflow, which already has Intel acceleration
features enabled, instead of the pip one
* add a batch size argument

### CODE REVIEW

**Comment by flaviodsr** [comment]
Very nice! 👏 

Just wondering if you should add the `batch_size` parameter when the doc says:

```
Integer or None. Number of samples per batch of computation. If unspecified, batch_size will default to 32. Do not
specify the batch_size if your data is in the form of a dataset, generators, or keras.utils.Sequence instances (since
they generate batches).
```
https://www.tensorflow.org/api_docs/python/tf/keras/Sequential

**Comment by jsuchome** [comment]
It's in conflict with https://github.com/fuseml/examples/pull/24 which seemed to be needed based on some previous tests...

File: codesets/mlflow/keras/train.py [hunk 1]
```diff
@@ -5,14 +5,34 @@
...
+        else:
+            from tensorflow.python.util import _pywrap_util_port
+            onednn_enabled = int(os.environ.get('TF_ENABLE_ONEDNN_OPTS', '0'))
+        mkl_enabled = _pywrap_util_port.IsMklEnabled() or (onednn_enabled == 1)
+    else:
+        mkl_enabled = tf.pywrap_tensorflow.IsMklEnabled()
+    return mkl_enabled
+
+print ("We are using Tensorflow version", tf.__version__)
+print("MKL enabled :", get_mkl_enabled_flag())
```
Comment (jsuchome) [inline codesets/mlflow/keras/train.py:28]: I assume this code is coming from the intel example pages of how to test the MKL enablement: could you add at least a comment to the function recognizing the source?

Also, this is really tiny bit, but could  you align the whitespace for those print methods on lines 27-28 ?

**Comment by stefannica** [comment]
> Very nice! clap
> 
> Just wondering if you should add the `batch_size` parameter when the doc says:
> 
> ```
> Integer or None. Number of samples per batch of computation. If unspecified, batch_size will default to 32. Do not
> specify the batch_size if your data is in the form of a dataset, generators, or keras.utils.Sequence instances (since
> they generate batches).
> ```
> 
> https://www.tensorflow.org/api_docs/python/tf/keras/Sequential

In our case, the data is a NumPy array, as returned by the `tensorflow.keras.datasets.cifar10.load_data()` function.

File: codesets/mlflow/keras/train.py (hunk 1 above)
Comment (stefannica) [reply codesets/mlflow/keras/train.py:35]: done

**Review by jsuchome — APPROVED** [review]
